src/components/data-table/table-batch-actions.ts

```diff
@@ -47,6 +47,12 @@ class BXTableBatchActions extends LitElement {
   @property({ type: Number, attribute: 'selected-rows-count' })
   selectedRowsCount = 0;
 
+  updated(changedProperties) {
+    if (changedProperties.has('active')) {
+      this.setAttribute('tabindex', `${this.active ? 0 : -1}`);
+    }
+  }
```

Comment:
Would you apply this roving `tabIndex` pattern to the button in Shadow DOM instead of the host element?

Reply:
Hm, I tried applying it to the button and any other divs that could be focused, but it seems it'd have to be applied to the host in order for it to work. It seems that each of the slotted `bx-btn` load without the default `tabindex`, making them able to be selected even if all the other elements have `tabindex="-1"`

Just as an FYI, the reason for this change is that in the default Sortable storybook variation, tabbing into the component makes the current bar go away, just like in the original issue that was opened, except with the opposite bar.